python/perspective/scripts/build_sdist.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
```

Comment:
(Quietly adds `bash` to the _smorgasbord_ of perspective dependency languages)

Reply:
its already there, notice this is not the first script in that directory